### PR TITLE
Add Storybook backgrounds addon

### PR DESCRIPTION
**Pull request by robinmetral**
## Purpose

Add the Storybook backgrounds addon, which adds an action on top of the preview to toggle the background between preset themes (and also a "toggle grid" action).

Screenshot 2022-02-03 at 10 51 24

This is going to make our testing easier for trying out theming (see #871).

## Approach and changes

- add the addon
- configure two backgrounds, light (`bodyBg`) and dark (`n900`)
- bump SB deps

## Definition of done

* [ ] Development completed
* [ ] Reviewers assigned
* [ ] Unit and integration tests
* [ ] Meets minimum browser support
* [ ] Meets accessibility requirements

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,19 +41,21 @@
...
+    "@storybook/addon-viewport": "^6.4.18",
+    "@storybook/addons": "^6.4.18",
+    "@storybook/builder-webpack5": "^6.4.18",
+    "@storybook/manager-webpack5": "^6.4.18",
     "@storybook/preset-typescript": "^3.0.0",
-    "@storybook/react": "^6.4.15",
-    "@storybook/source-loader": "^6.4.15",
+    "@storybook/react": "^6.4.18",
+    "@storybook/source-loader": "^6.4.18",
+    "@storybook/theming": "^6.4.18",
```
Comment (robinmetral) [inline package.json:58]: We were already using this dependency but hadn't explicitly installed it

**Review by connor-baer — APPROVED** [review]
Did you take a look at https://storybook.js.org/addons/storybook-dark-mode as well?

**Comment by connor-baer** [comment]
The grid toggle doesn't seem to work in the Vercel preview.

**Comment by robinmetral** [comment]
Yes, this will be useful to theme the manager when we actually land dark mode. However the setup is a bit more complicated so for now I'd go with the background only—it's enough for our testing for the abstraction layer

**Comment by robinmetral** [comment]
> The grid toggle doesn't seem to work in the Vercel preview.

I think it works but not if you change the background before 😩  since it's not really what we're looking for I'd just ignore it, is that fine with you?

**Review by amelako — APPROVED** [review]

**Comment by robinmetral** [comment]
Follow-up: I disabled the grid in 8f70885 (as mentioned, we don't need it, and it clutters the interface)

**Pull request merged**